src/orchestrator-process/src/lib.rs

```diff
@@ -628,13 +613,47 @@ impl OrchestratorWorker {
             None
         };
 
+        // The service might already exist. If it has the same config as requested (currently we
+        // can check only the scale), we have nothing to do. Otherwise we need to drop and recreate
+        // it.
```

Comment:
This is a functional change: Before we would simply leave existing processes unchanged and just add/drop ones according to the requested `scale`. I'm not sure if that approach made sense before, but it definitely doesn't make sense in a world where each process needs to start up with a compatible Timely config.